homeassistant/components/group/translations/en.json

```diff
@@ -40,6 +40,15 @@
                     "title": "New Group"
                 }
             },
+            "switch": {
+                "data": {
+                    "all": "All entities",
+                    "entities": "Members",
+                    "name": "Name",
+                    "title": "New Group"
+                },
+                "description": "If \"all entities\" is enabled, the group's state is on only if all members are on. If \"all entities\" is disabled, the group's state is on if any member is on."
```

Comment:
`all` feels like a weird option for a switch. Should we mark this only visible for advanced users? 

I also noted that we show it as part of the creation of a group. It seems like something that slim of a use case that we don't have to show it during creation, but allow users in advanced mode to enable it in the options later.

Reply:
Done!